packages/wasm/crate/src/tx.rs

```diff
@@ -349,6 +351,12 @@ pub async fn transaction_info_inner(
                 asset_ids.insert(action.description.output_id);
                 asset_ids.insert(action.description.input.asset_id);
             }
+            ActionView::ActionDutchAuctionWithdraw(ActionDutchAuctionWithdrawView {
+                reserves,
+                ..
+            }) => reserves.iter().for_each(|reserve| {
+                asset_ids.insert(reserve.asset_id());
+            }),
```

Comment:
This doesn't do anything yet, since core currently creates a stubbed withdraw view with [an empty reserves array](https://github.com/penumbra-zone/penumbra/blob/main/crates/core/transaction/src/is_action.rs#L500).

This is a [known issue](https://github.com/penumbra-zone/penumbra/issues/4219#issuecomment-2088801880) and will eventually be fixed. In the meantime, though, I'm writing the code here that will automatically populate the necessary metadata once core is updated.

Reply:
flagging that I'll also be inspecting these views closely per https://github.com/penumbra-zone/web/issues/1040, possibly changing what we display in our TxP / TxV

Reply:
Yeah it's hardcoded for now, my sense is that we can get to it a bit later, not next week but the one after, let me know if you think this should be prioritized and i'll get to it sooner